### PR TITLE
DDT-152: Add EquipmentEventTypeCode to EventSubscription

### PR DESCRIPTION
**Pull request by kagemand-andersen**

### CODE REVIEW

File: src/main/java/org/dcsa/tnt/model/EventSubscription.java [hunk 1]
```diff
@@ -13,7 +13,7 @@
 @Table("event_subscription")
 @Data
 @EqualsAndHashCode(callSuper = true)
-public class EventSubscription extends AbstractEventSubscription implements EventSubscriptionState {
+public class  EventSubscription extends AbstractEventSubscription implements EventSubscriptionState {
```
Comment (HenrikHL) [inline src/main/java/org/dcsa/tnt/model/EventSubscription.java:16]: suggestion```
public class EventSubscription extends AbstractEventSubscription implements EventSubscriptionState {
```

Comment (HenrikHL) [inline src/main/java/org/dcsa/tnt/model/EventSubscription.java:16]:
```suggestion
public class  EventSubscription extends AbstractEventSubscription implements EventSubscriptionState {
```

**Comment by HenrikHL** [comment]
Was this the only field that was missing?

**Comment by kagemand-andersen** [comment]
> Was this the only field that was missing?

Yes.

**Review by HenrikHL — APPROVED** [review]

**Pull request merged**